SOC/common/simulation.py

```diff
@@ -270,8 +318,17 @@ def get_exponent(self,
         print(f"y = {10**coef_b:.3f} exp({coef_a:.4f} x)")
         return dict(exponent=coef_a, intercept = coef_b)
 
+    # TODO how is this different from `load`?
```

Comment:
Próbowałem dziś wczytać z zapisanej przez zarr grupy, ale nie udało mi się (nie byłem w stanie dotrzeć do danych, chociaż widziałem strukturę w Grupy). W pewnej swobodniejszej przyszłości wrócę do tego.:)

Reply:
(Natomiast, tak jak teraz doświadczam, samo bieżące zapisywanie działa i ratuje te dłuższe symulacje przed zbraknięciem pamięci. W sumie mega.)

Reply:
To pewnie jeszcze trzeba będzie poprawić. Wersja 1.1? ;)